projects/material-css-vars/src/lib/_mat-lib-overwrites.scss

```diff
@@ -1,143 +1,377 @@
-@use "variables" as variables;
-@use "public-util" as utilities;
+@use 'public-util';
+@use "variables";
 
-// main overwrite to make it all work
-@function get-color-from-palette($palette, $hue: default, $opacity: null) {
-
-  @if type-of($hue) == number and $hue >= 0 and $hue <= 1 {
-    @return get-color-from-palette($palette, default, $hue);
-  }
-
-  @return map-get($palette, $hue);
-}
-
-// apply additional overwrites
+// ---------------------------
+// MDC Overwrites
+// ---------------------------
 @mixin other-overwrites {
-  @include _mat-date-picker-content-overwrites();
```

Comment:
The date picker was not affected by the MDC update, need to check whether still need this overwrites